File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/recyclerView/selector/MeteoriteSelectorLandscape.java
package com.antonio.samir.meteoritelandingsspots.ui.recyclerView.selector;


public class MeteoriteSelectorLandscape implements MeteoriteSelector {

    private final MeteoriteSelectorView view;

    public MeteoriteSelectorLandscape(MeteoriteSelectorView view) {
        this.view = view;
    }

    @Override
    public void selectItemId(final String meteorite) {
        view.selectLandscape(meteorite);
    }

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/recyclerView/selector/MeteoriteSelectorLandscape.kt
package com.antonio.samir.meteoritelandingsspots.ui.recyclerView.selector


class MeteoriteSelectorLandscape(private val view: MeteoriteSelectorView) : MeteoriteSelector {

    override fun selectItemId(meteorite: String) {
        view.selectLandscape(meteorite)
    }

}
